pkg/controller/issuers/sync.go

```diff
@@ -27,7 +27,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/errors"
 
 	apiutil "github.com/jetstack/cert-manager/pkg/api/util"
-	"github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
+	v1 "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
```

Comment:
nit: we don't need this alias (I think this is your IDE being 'helpful' 😬)

Reply:
It's `goimports` being helpful. Perhaps it's doing the right thing according to this issue: https://github.com/golang/go/issues/29041 I can't tell.
But I've removed it anyway.
The convention elsewhere in cert-manager seems to be to use `import cmapi "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"` when importing the default version of the API, but I won't change that here.